src/main/java/lotto/Buyer.java

```diff
@@ -1,23 +1,27 @@
 package lotto;
 
+import java.math.BigDecimal;
+
 public class Buyer {
     private static final String PURCHASE_EXCEPTION_MESSAGE = "구입금액은 1000원 이상의 금액을 " +
             "입력해야 합니다.";
 
     private LottoPrice lottoPrice = new LottoPrice();
     private int purchase;
+    private BigDecimal purchaseDecimal;
 
     public Buyer(int purchase) {
         validatePurchase(purchase);
         this.purchase = purchase;
     }
 
     public int buyLotto() {
-        return this.purchase / lottoPrice.getLottoPrice();
+        purchaseDecimal = new BigDecimal(this.purchase);
```

Comment:
```suggestion
        purchaseDecimal = BigDecimal.valueOf(purchase);
```

new 연산자를 통해 생성도 좋지만 BigDecimal은 static 메서드릴 지원하고 있기 때문에
위와 같이 사용할 수 도 있습니다!! 참고해주세요 😄